srcpkgs/php-meta/template

```diff
@@ -0,0 +1,173 @@
+# Template file for 'php-meta'
+pkgname=php-meta
```

Comment:
hmm, I think this will result in having broken links if only `php<version>` is installed, but not the subpackages that actually provide the files mentioned in alternatives

Reply:
Yes it does.

Reply:
I don't see a straightforward way to handle this with our current alternatives system, only some workarounds:

We could let the php package provide the php alternative that would point /usr/bin/php -> php8.{0,1}. All other unversioned binaries (php-fpm, composer, etc) would be symlinks pointing to a multicall script called php-alternatives provided by the php package. This script would look at /usr/bin/php, detect the version from there and add the version suffix to its arg 0.

What do you think about this?

Reply:
It seems like a workable solution except for the man pages--I'm not seeing a good way to make that fit with your proposal. 

Other options:
1. Have a single alternatives group and install all the packages in that group as dependencies of the base package (e.g. `php8.1-devel php8.1-phpdbg php8.1-fpm php8.1-pear php8.1-cgi`) This would bring a PHP 8.1 install from 12MB on disk to 74MB on disk.
2. Don't provide alternatives for anything except the core binaries available in the base package (`php`, `phar`, and `phar.phar`). This would meet most use cases where the user just wants `php` available in path for convenience or hardcoded calls to `php` in some script.

Reply:
I'm opting for my option 2.
regarding option 1: this is inelegant.

Regarding @paper42's proposal: I don't think it would be good to have binaries available that don't have a manpage tied to them.

Reply:
Any input on this? Hoping to get this in so I can get 8.2 in repo.

Reply:
I didn't forget about this PR, but I didn't have the mental capacity for this lately. I plan to look into this soon.

Reply:
I think we can go with the second option, it's relatively elegant and simple.

Reply:
OK thanks for the input! I have updated this PR to include it and fixed merge conflicts.